imports/api/crud-controller/module/methods.js

```diff
@@ -160,3 +160,34 @@ export const deleteOneModuleInSemester = function deleteOneModuleInSemester(seme
 
   return deletedModule;
 };
+
+export const deleteAllModulesInSemester = function deleteAllModulesInSemester(semesterIndex, plannerID) {
```

Comment:
This is for clearing a semester's worth of modules. Similar to CRUD implementation of other functions too, noted and will be working on "Clear semester's worth of modules" for UI soon! :)